site/src/pages/hub.page.tsx

```diff
@@ -27,14 +30,14 @@ export const getStaticProps: GetStaticProps<PageProps> = async () => {
 const HubPage: NextPage<PageProps> = ({ catalog }) => {
   return (
     <>
-      <Head>
-        <title>Block Protocol – Block Hub</title>
-        <meta itemProp="name" content="Block Hub" />
-        <meta
-          itemProp="description"
-          content="The Block Protocol's registry of open-source blocks"
-        />
-      </Head>
+      <NextSeo
+        title={seoTitle}
+        description={seoDesc}
+        openGraph={{
+          description: seoDesc,
+          title: seoTitle,
+        }}
+      />
```

Comment:
I searched for a reason to keep `itemProp="name"` and couldn't find any. Let me know if I missed sth and openGraph tags are not enough here.